examples/storage/minio-distributed/README.md

```diff
@@ -0,0 +1,195 @@
+# Cloud Native Deployment of Distributed Minio using Kubernetes
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Quickstart](#quickstart)
+- [Step 1: Create Minio Headless Service](#step-1-create-minio-headless-service)
+- [Step 2: Create Minio Statefulset](#step-2-create-minio-statefulset)
+- [Step 3: Create LoadBalancer Service](#step-3-create-minio-service)
+- [Step 4: Resource cleanup](#step-4-resource-cleanup)
+
+## Introduction
+Minio is an AWS S3 compatible, object storage server built for cloud applications and devops. Minio is _cloud native_, meaning Minio understands that it
+is running within a cluster manager, and uses the cluster management infrastructure for allocation of compute and storage resources.
+
+The following document describes the process to deploy [distributed Minio](https://docs.minio.io/docs/distributed-minio-quickstart-guide) server on Kubernetes.
+This example uses the [official Minio Docker image](https://hub.docker.com/r/minio/minio/~/dockerfile/) from Docker Hub.
+
+This example uses some of the core components of Kubernetes:
+
+- [_Pods_](https://kubernetes.io/docs/user-guide/pods/)
+- [_Services_](https://kubernetes.io/docs/user-guide/services/)
+- [_Statefulsets_](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/)
+
+## Prerequisites
+
+This example assumes that you have a Kubernetes version >=1.5 cluster installed and running, and that you have installed the [`kubectl`](../../../docs/user-guide/kubectl/kubectl.md)
+command line tool somewhere in your path.  Please see the
+[getting started guides](../../../docs/getting-started-guides/)
+for installation instructions for your platform.
+
+## Quickstart
+
+Run the below commands to get started quickly
+
+```sh
+kubectl create -f examples/storage/minio/minio-distributed-headless-service.yaml
```

Comment:
It may be an idea to actually state where you can get these files. For those who are reading on GitHub and have 1.5.2 release so won't have these files.